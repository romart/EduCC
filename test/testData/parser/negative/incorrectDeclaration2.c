
/ TODO



int []ff {

}

int foo,() {

}
