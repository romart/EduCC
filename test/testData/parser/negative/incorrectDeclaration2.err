test/testData/parser/negative/incorrectDeclaration2.c:1:2: warning: type specifier missing, defaults to 'int'
test/testData/parser/negative/incorrectDeclaration2.c:1:2: warning: declaration does not declare anything
test/testData/parser/negative/incorrectDeclaration2.c:1:2: error: unexpected token / '/' instead of {
test/testData/parser/negative/incorrectDeclaration2.c:3:2: error: use of undeclared identifier 'TODO'
test/testData/parser/negative/incorrectDeclaration2.c:1:6: error: unexpected token INT 'int' instead of ;
test/testData/parser/negative/incorrectDeclaration2.c:5:6: error: unexpected token [ '[' instead of (
test/testData/parser/negative/incorrectDeclaration2.c:6:6: error: unexpected token ] ']' instead of (
test/testData/parser/negative/incorrectDeclaration2.c:7:6: error: use of undeclared identifier 'ff'
test/testData/parser/negative/incorrectDeclaration2.c:10:6: error: unexpected token { '{' instead of )
test/testData/parser/negative/incorrectDeclaration2.c:1:8: error: unexpected token } '}' instead of )
test/testData/parser/negative/incorrectDeclaration2.c:1:10: error: unexpected token INT 'int' instead of ;
test/testData/parser/negative/incorrectDeclaration2.c:5:10: error: use of undeclared identifier 'foo'
test/testData/parser/negative/incorrectDeclaration2.c:8:10: error: unexpected token , ',' instead of (
test/testData/parser/negative/incorrectDeclaration2.c:10:10: error: unexpected token ) ')' instead of (
test/testData/parser/negative/incorrectDeclaration2.c:12:10: error: unexpected token { '{' instead of (
test/testData/parser/negative/incorrectDeclaration2.c:1:12: error: unexpected token } '}' instead of (
