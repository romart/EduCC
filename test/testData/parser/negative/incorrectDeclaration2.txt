FILE test/testData/parser/negative/incorrectDeclaration2.c
  signed int