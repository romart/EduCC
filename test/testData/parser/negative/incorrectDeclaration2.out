test/testData/parser/negative/incorrectDeclaration2.c:1:2: warning: type specifier missing, defaults to 'int'
test/testData/parser/negative/incorrectDeclaration2.c:1:2: warning: declaration does not declare anything
